sdk/helper/testcluster/docker/environment.go

```diff
@@ -726,7 +728,9 @@ func (n *DockerClusterNode) start(ctx context.Context, caDir string, opts *Docke
 		}
 	}
 	n.ContainerNetworkName = netName
+	n.ContainerID = svc.Container.ID
 	n.ContainerIPAddress = svc.Container.NetworkSettings.Networks[netName].IPAddress
+	n.ExposedHostPort = svc.Container.NetworkSettings.Ports["8200/tcp"][0].HostPort
```

Comment:
How is this different from HostPort?

Reply:
It didn't include the IP address, but will remove/roll this back as well as that is easy enough to remove. Thanks for the feedback!